src/content/docs/cypher/data-definition/alter.md

```diff
@@ -66,6 +66,47 @@ This query tells Kuzu to only drop the `grade` column if it exists.
 
 The same applies to relationship tables.
 
+## Add From-To Node Table Connection
+
+`ADD FROM <node_table_name> TO <node_table_name>` allows you to add a connection between two node tables into an existing relationship table.
+
+The following example creates a node table `Celebrity` and adds `User` follows `Celebrity` into `Follows` relationship table.
+```sql
+CREATE NODE TABLE Celebrity(name STRING PRIMARY KEY);
+ALTER TABLE Follows ADD FROM User TO Celebrity;
+```
+
+#### Add From-To Node Table Connection if not exists
```

Comment:
These sections are repeated 4 times in this page. How about we describe it only once in the first one and for the remaining sections, add something like the following to the previous section:
```
You can also use the `IF NOT EXISTS` modifier to do nothing if the given connection already exists.
ALTER TABLE Follows ADD IF NOT EXISTS FROM User TO Celebrity;
```